x-pack/plugins/apm/server/routes/rum_client/route.ts

```diff
@@ -190,22 +189,6 @@ const rumPageViewsTrendRoute = createApmServerRoute({
   },
 });
 
-const rumServicesRoute = createApmServerRoute({
-  endpoint: 'GET /internal/apm/ux/services',
-  params: t.type({
-    query: t.intersection([uiFiltersRt, rangeRt]),
-  }),
-  options: { tags: ['access:apm'] },
-  handler: async (resources): Promise<{ rumServices: string[] }> => {
-    const setup = await setupUXRequest(resources);
-    const {
-      query: { start, end },
-    } = resources.params;
-    const rumServices = await getRumServices({ setup, start, end });
-    return { rumServices };
-  },
-});
-
 const rumVisitorsBreakdownRoute = createApmServerRoute({
```

Comment:
I assume the remaining routes will have to be moved out in a future PR?

Reply:
@sqren yes we are going to be doing this one component at a time, or grouped if there are some that it makes sense to move at once. Some are going to be ported to using data plugin queries, others will become embeddables, with the goal of avoiding adding a server component to the UX plugin. We're planning to have them all removed soon!